cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -882,5 +882,6 @@
     public static final String EXEC_THREADS = "metadata.service.exec.threads";
     public static final String METADATA_HANDLERS_NAME = "metadata.handlers";
     public static final String METADATA_UPDATES_KAFKA_TOPIC = "metadata.updates.kafka.topic";
+    public static final String METADATA_UPDATES_PUBLISH_ENABLED = "metadata.updates.publish.enabled";
```

Comment:
Also, for the `EXEC_THREADS`, it's better name as `SERVICE_EXEC_THREADS` so that when used in code, it's fluently read as `Constants.Metadata.SERVICE_EXEC_THREADS` which matches the key name in cConf pretty well.